### PR TITLE
Get rid of spaces.

### PR DESCRIPTION
**Pull request by profi200**
Ok, again. This replaces only spaces for indentation. Instead of just closing pull requests look what changed. I'm not wasting my time for a few people with stone age editors to close my pull request instead of beeing happy that someone is trying to polish things/contribute.

### CODE REVIEW

File: libctru/include/3ds/svc.h [hunk 1]
```diff
@@ -24,14 +24,14 @@ typedef enum {
...
-    u32 state;
+		u32 base_addr;
+		u32 size;
+		u32 perm;
+		u32 state;
 } MemInfo;
 
 typedef struct {
-    u32 flags;
+		u32 flags;
```
Comment (mtheall) [inline libctru/include/3ds/svc.h]: Use one tab here instead of two.

File: libctru/include/3ds/svc.h [hunk 2]
```diff
@@ -24,14 +24,14 @@ typedef enum {
...
 
 typedef struct {
-    u32 base_addr;
-    u32 size;
-    u32 perm;
-    u32 state;
+		u32 base_addr;
+		u32 size;
+		u32 perm;
+		u32 state;
```
Comment (mtheall) [inline libctru/include/3ds/svc.h]: Use one tab here instead of two.

File: libctru/source/gfx.c [hunk 3]
```diff
@@ -31,17 +31,17 @@ void gfxSet3D(bool enable)
...
 
 void gfxSetScreenFormat(gfxScreen_t screen, GSP_FramebufferFormats format) {
-    if(screen==GFX_TOP)
-        topFormat = format;
-    else
-        botFormat = format;
+		if(screen==GFX_TOP)
+				topFormat = format;
+		else
+				botFormat = format;
```
Comment (mtheall) [inline libctru/source/gfx.c]: Too many tabs.

File: libctru/source/gfx.c [hunk 4]
```diff
@@ -31,17 +31,17 @@ void gfxSet3D(bool enable)
...
 
 GSP_FramebufferFormats gfxGetScreenFormat(gfxScreen_t screen) {
-    if(screen==GFX_TOP)
-        return topFormat;
-    else
-        return botFormat;
+		if(screen==GFX_TOP)
+				return topFormat;
+		else
+				return botFormat;
```
Comment (mtheall) [inline libctru/source/gfx.c]: Too many tabs.

File: libctru/source/gfx.c [hunk 5]
```diff
@@ -49,18 +49,18 @@ void gfxSetDoubleBuffering( gfxScreen_t screen, bool doubleBuffering) {
...
+				return 4;
+		case GSP_BGR8_OES:
+				return 3;
+		case GSP_RGB565_OES:
+		case GSP_RGB5_A1_OES:
+		case GSP_RGBA4_OES:
+				return 2;
+		}
+
+		return 3;
```
Comment (mtheall) [inline libctru/source/gfx.c]: Too many tabs.

File: libctru/source/services/httpc.c [hunk 6]
```diff
@@ -39,14 +39,14 @@ Result httpcOpenContext(httpcContext *context, char* url, u32 use_defaultproxy)
 	if(ret!=0) {
 		HTTPC_CloseContext(__httpc_servhandle, context->httphandle);
 		return ret;
-        }
+				}
```
Comment (mtheall) [inline libctru/source/services/httpc.c]: This should just be one tab.

File: libctru/source/services/httpc.c [hunk 7]
```diff
@@ -39,14 +39,14 @@ Result httpcOpenContext(httpcContext *context, char* url, u32 use_defaultproxy)
...
 
 	ret = HTTPC_InitializeConnectionSession(context->servhandle, context->httphandle);
 	if(ret!=0) {
 		HTTPC_CloseContext(__httpc_servhandle, context->httphandle);
 		svcCloseHandle(context->servhandle);
 		return ret;
-        }
+				}
```
Comment (mtheall) [inline libctru/source/services/httpc.c]: This should just be one tab.

File: libctru/source/services/httpc.c [hunk 8]
```diff
@@ -55,7 +55,7 @@ Result httpcOpenContext(httpcContext *context, char* url, u32 use_defaultproxy)
...
```
Comment (mtheall) [inline libctru/source/services/httpc.c]: This should just be one tab.

File: libctru/source/srv.c [hunk 9]
```diff
@@ -9,14 +9,14 @@
...
-  find the service in that list, we ask the service manager and hope for the
-  best.
+	The homebrew loader can choose to supply a list of service handles that have
+	been "stolen" from other processes that have been compromised. This allows us
+	to access services that are normally restricted from the current process.
+
+	For every service requested by the application, we shall first check if the
+	list given to us contains the requested service and if so use it. If we don't
+	find the service in that list, we ask the service manager and hope for the
+	best.
```
Comment (mtheall) [inline libctru/source/srv.c]: Are these tabs necessary?

File: libctru/source/srv.c [hunk 10]
```diff
@@ -116,7 +116,7 @@ Result srvGetServiceHandle(Handle* out, const char* name)
 	Result rc = 0;
 
 	/* Look in service-list given to us by loader. If we find find a match,
-	   we return it. */
+     we return it. */
```
Comment (mtheall) [inline libctru/source/srv.c]: Revert this.

File: libctru/source/services/httpc.c (hunk 8 above)
Comment (mtheall) [inline libctru/source/services/httpc.c]: This should just be one tab.

**Comment by profi200** [comment]
All done.

**Comment by WinterMute** [comment]
Sorry, it went stale.

17 file patch with 898 changed lines is too much for whitespace changes

**Pull request closed without merging**